components/NavMenu.js

```diff
@@ -0,0 +1,65 @@
+import Link from "next/link";
+import { useState } from "react";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+export default function NavMenu() {
+  const { pathname } = useRouter();
+
+  const [hamburgerMenu, setHamburgerMenu] = useState(false);
```

Comment:
The naming of this state could be more descriptive (e.g. showMenu)

Reply:
renamed state to: showHamburgerMenu